src/Alarm.cpp

```diff
@@ -53,91 +80,138 @@ void Alarm::determineBasePhotoresistorReading(){
     _greenLED->flash();
   }
   _baseReading = avgReading / numOfReadings;
-  Logger::Log("Base Photoresistor Reading Determined");
+  Serial.write(13101); // 13101 = Log, Debug, Alarm base photorsistor reading determined
   _laser->on();
 }
-
+/**
+  Makes the buzzer produce the Negative tone
+*/
 void Alarm::alertFailedAction(){
-  Logger::Log("Alarm Failed Action");
+  Serial.write(13102); // 13102 = Log, Debug, Alarm failed action
   _buzzer->soundNegativeTone();
   _redLED->flash(1000);
 }
-
+/**
+  Makes the buzzer produce the Affirmative tone
+*/
 void Alarm::alertSuccessfulAction(){
-  Logger::Log("Alarm successful action");
+  Serial.write(13103); // 13103 = Log, Debug, Alarm successful action
   _buzzer->soundAffirmativeTone();
   _greenLED->flash(1000);
 }
-
+/**
+  Turns the LASER on and if it fails isReadyToArm turns it back off to as it the
+    Alarm class needs to be calibrated again and leaves it on and sets isArmed
+    to true and the _alarmLED to be on.
+  @param _laser turns it on or keeps it off if calibration failed
+  @param _alarmLED if alertSuccessfulAction is turned on
+  @param _isArmed is alertSuccessfulAction is set to true
+*/
 void Alarm::arm(){
+  Serial.write(30001); // 30001 = Command, N/A, Execute Command: Arm
   _laser->on();
   if (!this->isReadyToArm()){
-   Logger::Log("Alarm failed to arm");
+   Serial.write(13104); // 13104 = Log, Debug, Alarm failed to arm
    this->alertFailedAction();
    if(!this->isArmed()) _laser->off();
  } else {
    this->alertSuccessfulAction();
-    Logger::Log("Alarm sucessfully armed");
+    Serial.write(13105); // 13105 = Log, Debug, Alarm successfully armed
    _alarmLED->on();
    _isArmed = true;
  }
 }
-
+/**
+  @return returns true if the isArmed is not armed and isCalibrated is
+    calibrated, and isTripped and isTriggered are not activated
+*/
 bool Alarm::isReadyToArm(){
   return(!this->isArmed()
         && this->isCalibrated()
         && !this->isTripped()
         && !this->isTriggered());
 }
-
+/**
+  Getter for the param _isArmed
+*/
 bool Alarm::isArmed(){
   return _isArmed;
 }
-
+/**
+  @return returns a true false based on whether or the reading taken is less
+    than the _baseReading by 100
+*/
 bool Alarm::isTripped(){
   return (_photoR->takeReading() - 100 < _baseReading);
 }
-
+/**
+  Sets the alarm to be triggered
+  @param sets _isTriggered to be true
+  @param sets _isSilenced to be false
+*/
 void Alarm::trigger(){
-  Logger::Log("Alarm has been triggered!");
+  Serial.write(30005); // 30005 = Command, N/A, Execute Command: Trigger
```

Comment:
Again, 000 - 099 are for commands that the PI sends back to the arduino to tell it to do something.